ui/src/message_popup/message_popup_page.jsx

```diff
@@ -138,26 +140,7 @@ export default React.createClass({
         <p style={styles.paragraph}>You may be asked to write, sketch or say your responses aloud.</p>
         <p style={styles.paragraph}>Each question is timed to simulate responding in the moment in the classroom.  You'll have 60 seconds to respond to each question.</p>
         <Divider />
-        <div style={{}}>
-          <div style={{fontSize: 16, padding: 20}}>
-            <Toggle
-              label="With student cards"
-              labelPosition="right"
-              toggled={this.state.shouldShowStudentCards}
-              onToggle={this.onStudentCardsToggled} />
-            <Toggle
-              label="With feeback and revision"
-              labelPosition="right"
-              toggled={false}
-              disabled={true}  />
-            <Toggle
-              label="With hints available"
-              labelPosition="right"
-              toggled={false}
-              disabled={true} />
```

Comment:
Everything looks great and moving this to its own render function is a great idea. Only problem I see is that it might have a conflict with my PR because I have something changed in here (a very small change, I had `disabled={true}` removed and changed the `toggled` and `onToggle` parts for the "With hints available" Toggle. Its not an issue but I will need help with merging the pull request.